packages/calcite-components/src/components/dialog/dialog.tsx

```diff
@@ -366,6 +375,11 @@ export class Dialog extends LitElement implements OpenCloseComponent, LoadableCo
 
   onOpen(): void {
     this.calciteDialogOpen.emit();
+
+    if (!this.open) {
```

Comment:
Is this necessary? We don't prevent activation in other focus-trapping components.

Reply:
This shouldn't be necessary here.